debugtools/DDR_VM/test.xml

```diff
@@ -81,7 +81,7 @@ TCK - runs the DTFJ TCK
 			<entry key="tck.dumpcreator.class" value="ibm.j9.dumpcreators.SystemDumpCreator"/>
 			<entry key="tck.imagecreator.class" value="com.ibm.j9ddr.dtfj.tck.ImageCreator"/>
 			<entry key="tck.dumpcreator.launcher" value="${sdk.executable}"/>
-			<entry key="env.IBM_JAVA_OPTIONS" value="-Xdump:system:defaults:file=&quot;${tck.core.file}&quot;"/>
+			<entry key="env.OPENJ9_JAVA_OPTIONS" value="-Xdump:system:defaults:file=&quot;${tck.core.file}&quot;"/>
```

Comment:
Change is still not reverted

Reply:
Working on it.  This is still WIP; I will let you know when it's ready for another review.

Reply:
I see you've removed the WIP but this change hasn't been reverted yet.